provider/pkg/gen/schema.go

```diff
@@ -78,7 +78,7 @@ func PulumiSchema(swagger map[string]interface{}) pschema.PackageSpec {
 					DeprecationMessage: "This option has been replaced by `enableServerSideApply`.",
 				},
 				"enableServerSideApply": {
-					Description: "BETA FEATURE - If present and set to true, enable Server-Side Apply mode.\nSee https://github.com/pulumi/pulumi-kubernetes/issues/2011 for additional details.\nThis feature is in developer preview, and is disabled by default.",
+					Description: "If present and set to true, enable Server-Side Apply mode.\nSee https://github.com/pulumi/pulumi-kubernetes/issues/2011 for additional details.",
```

Comment:
Shouldn't this be the other way around, if it defaults to true? As in,

> If present at set to false, Server-Side Apply mode is disabled. [etc...]

Reply:
Good catch, fixed!